appshell/appshell_extensions.cpp

```diff
@@ -606,6 +606,9 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
                 bool enabled = argList->GetBool(2);
                 bool checked = argList->GetBool(3);
                 error = NativeMenuModel::getInstance(getMenuParent(browser)).setMenuItemState(command, enabled, checked);
+                if (error == NO_ERROR) {
+                    error = SetMenuItemState(browser, command, enabled, checked);
```

Comment:
this seems to be common code (i.e. windows, mac and win). 
Isn't the menu item state being set  in line 608?